spec/02-integration/05-proxy/10-server_tokens_spec.lua

```diff
@@ -0,0 +1,286 @@
+local helpers = require "spec.helpers"
+local constants = require "kong.constants"
+
+
+local default_server_header = _KONG._NAME.."/".._KONG._VERSION
+
+
+local function start(config)
+  return function()
+    helpers.dao.apis:insert {
+      name = "api-1",
+      upstream_url = "http://localhost:9999/headers-inspect",
+      hosts = "inexistent.com",
```

Comment:
It's really confusing to have this value set to `inexistent.com`. Especially when also using `404.com` in the tests. How about `header-inspect`? Also, this value should be a table for consistency's sake and potentially forward compatibility.